### PR TITLE
Fix issues found by md-nits

### PR DESCRIPTION
**Pull request by richsalz**
Trivial stuff.  md-nits should be added to the CI (#15459)

### CODE REVIEW

**Review by t8m — APPROVED** [review]

File: README-PROVIDERS.md [hunk 1]
```diff
@@ -83,11 +81,9 @@ automatically loaded, the null provider can be loaded instead.
 This can be useful if you are using non-default library contexts and want
 to ensure that the default library context is never used unintentionally.
 
-
 Loading Providers
 =================
 
-
```
Comment (slontis) [inline README-PROVIDERS.md:90]: @levitte - is there any reason for these '-' ?

Comment (richsalz) [reply README-PROVIDERS.md:90]: Are you asking about dashes (used to make bullet lists, at the top of the file for a table of contents) or misreading my extra blank line deletion, or ask if they were there for a reason?

Comment (slontis) [reply README-PROVIDERS.md:90]: Was there a reason.

Comment (t8m) [reply README-PROVIDERS.md:90]: @slontis please look again, this is just a deletion of an extra blank line. IMO a correct change.

**Review by slontis — APPROVED** [review]

**Comment by levitte** [comment]
Merged

3e3ad3c54855dd534437871a0c78858de5e3d246 Fix issues found by md-nits

**Pull request closed without merging**